draft-foudil-securitytxt.md

```diff
@@ -87,11 +87,11 @@ in disclosing security issues.
 The file is named "security.txt", and this file SHOULD be placed under the
 /.well-known/ path ("/.well-known/security.txt") {{!RFC5785}} of a domain name or IP address for web
 properties. If it is not possible to place the security.txt file in the /.well-known/ path or setup a redirect, web-based services MAY place the file in the top-level path
-of a given web domain or IP address ("/security.txt") as a fall back option. For web-based services, the file MUST be accessible via the Hypertext Transfer Protocol {{!RFC1945}} as a resource of Internet Media Type "text/plain" with the default charset parameter set to "utf-8" per section 4.1.3 of {{!RFC2046}}, and it is RECOMMENDED that this file be served with "https" (as per section 2.7.2 of {{!RFC7230}}). For file systems and version control repositories a ".security.txt" file SHOULD be placed in the root directory of a particular file system or source code project.
+of a given web domain or IP address ("/security.txt") as a fall back option. For web-based services, the file MUST be accessible via the Hypertext Transfer Protocol {{!RFC1945}} as a resource of Internet Media Type "text/plain" with the default charset parameter set to "utf-8" per section 4.1.3 of {{!RFC2046}}, and it is RECOMMENDED that this file be served with "https" (as per section 2.7.2 of {{!RFC7230}}. For file systems and version control repositories a ".security.txt" file SHOULD be placed in the root directory of a particular file system or source code project.
 
 This text file contains multiple directives
 with different values. The "directive" is the first part of a field all the way up
-to the colon ("Contact:"). Directives MUST be case-insensitive (as per section 2.3 of {{!RFC5234}}).
```

Comment:
Were these closing parenthesis meant to be removed?

Reply:
Sorry I was trying to fix something and messed up. I updated the pull request.